clash-ghc/src-ghc/Clash/GHC/Evaluator/Primitive.hs

```diff
@@ -880,7 +926,7 @@ ghcPrimStep tcm isSubj pInfo tys args mach = case primName pInfo of
     -> reduce r
   "GHC.Prim.xorWord32#" | Just r <- liftW32 xorWord32# args
     -> reduce r
-  "GHC.Prim.notWord32" | [i] <- word32Literals' args
+  "GHC.Prim.notWord32#" | [i] <- word32Literals' args
```

Comment:
Did the old names never exist? Why does this work on older versions of GHC?

Reply:
The old names never existed.